lnwallet/commitment.go

```diff
@@ -83,13 +83,20 @@ func DeriveCommitmentKeys(commitPoint *btcec.PublicKey,
 
 	tweaklessCommit := chanType.IsTweakless()
 
+	// Depending on if this is our commit or not, we'll choose the correct
+	// base point.
+	localBasePoint := localChanCfg.PaymentBasePoint
+	if isOurCommit {
+		localBasePoint = localChanCfg.DelayBasePoint
+	}
+
```

Comment:
Merge this with `if isOurCommit {` block below where the other base points are calculated? (bring everything up)

Reply:
Discussed offline, excluded from thispr